cosmos/operators/base.py

```diff
@@ -55,6 +55,7 @@ class DbtBaseOperator(BaseOperator):
     :param dbt_executable_path: Path to dbt executable can be used with venv
         (i.e. /home/astro/.pyenv/versions/dbt_venv/bin/dbt)
     :param dbt_cmd_flags: List of flags to pass to dbt command
+    :param dbt_cmd_global_flags: List of dbt global flags to be passed to the dbt command
```

Comment:
```suggestion
    :param dbt_global_flags: List of dbt global flags to be passed to the dbt command
```

Reply:
@harels we spoke about this and I talked to @jlaneve about this to get a third opinion on the naming, and he voted for us to keep the originally proposed - so I won't be applying the rename.